announcement/announcement-tool/tool/src/java/org/sakaiproject/announcement/tool/AnnouncementAction.java

```diff
@@ -1002,7 +1007,7 @@ else if (statusName.equals(VIEW_STATUS))
 		{
 			template = buildShowMetadataContext(portlet, context, rundata, state, sstate);
 		}
-		else if ((statusName.equals("goToReviseAnnouncement")) || (statusName.equals("backToReviseAnnouncement"))
+		else if ((statusName.equals("goToReviseAnnouncement")) || (statusName.equals(BACK_TO_EDIT_STATUS))
```

Comment:
I know this is already present, what do you think about substituting this line by:

 StringUtils.equalsAnyIgnoreCase(statusName, "goToReviseAnnouncement", BACK_TO_EDIT_STATUS, ADD_STATUS, "stayAtRevise")